docs/p2p-check.md

```diff
@@ -55,14 +54,24 @@ modals:
   }
 </style>
 
-{{ beta("feature") }}
+## Port Forwarding
+
+Peer-to-peer (P2P) connections generally allow for lower frames of delay with less lag than playing on server. However, P2P requires one player to host, and the host must have port forwarding enabled. Port forwarding is essentially telling your router that all traffic to your network with a certain identifier (the port) should be sent to your computer. 
+
+For hosting P2P, you'll need to forward **UDP traffic** on **port 27886**.
+
+In general, you'll need to navigate to your router's web address, log in using the admin credentials (often printed on the router), find the port forwarding section, and add a new rule to forward UDP port 27886 traffic to your computer.
+
+Because every router is different, these steps are very general. For more specific instructions, check your router's documentation, or search for port forwarding instructions with your router's model number.
+
+## P2P Checker
```

Comment:
After thinking about it, I'm mostly okay with these changes. The admonitions were too much and the way you removed them was fine.

My only requirement is I want the tool at the top of the page for now. These need to be moved/reorganized, but I can't really see a good way to handle this right now. I'm okay with either putting the `Port Forwarding` section below the checker, or creating a separate page entirely for port forwarding. One should probably exist anyway so it's fine.

As long as the P2P checker is at the top of the page, it's good enough for now.

Maybe we should change the page title.

Reply:
Moved the tool to the top, let me know if you think it would make more sense as 2 separate pages, I'm unsure